apps/crn-server/test/data-providers/contentful/analytics/collaboration.analytics.data-provider.test.ts

```diff
@@ -83,3 +93,455 @@ describe('collaboration', () => {
+      take: 1,
+    });
+
+    expect(result).toEqual({
+      total: 1,
+      items: [
+        expect.objectContaining({
+          outputsCoProducedWithin: {
+            Article: 0,
+            Bioinformatics: 0,
+            Dataset: 0,
+            'Lab Resource': 0,
+            Protocol: 0,
+          },
+          outputsCoProducedAcross: {
+            byDocumentType: {
+              Article: 0,
+              Bioinformatics: 0,
+              Dataset: 0,
+              'Lab Resource': 0,
+              Protocol: 0,
+            },
+            byTeam: [],
+          },
+        }),
+      ],
+    });
+  });
+  describe('within team', () => {
+    test('Should return a count of 2 if client has two outputs of the same document type with multiple labs', async () => {
+      const graphqlResponse = getTeamCollaborationQuery();
+      const researchOutputs = [
+        {
+          addedDate: '2023-09-01T03:00:00.000Z',
+          createdDate: '',
+          documentType: 'Article',
+          labsCollection: {
+            total: 3,
+          },
+          teamsCollection: {
+            items: [],
+          },
+        },
+        {
+          addedDate: '2023-08-30T03:00:00.000Z',
+          createdDate: '',
+          documentType: 'Article',
+          labsCollection: {
+            total: 3,
+          },
+          teamsCollection: {
+            items: [],
+          },
+        },
+        {
+          addedDate: '2023-09-01T03:00:00.000Z',
+          createdDate: '',
+          documentType: 'Protocol',
+          labsCollection: {
+            total: 1,
+          },
+          teamsCollection: {
+            items: [],
+          },
+        },
+        {
+          addedDate: '2023-07-01T03:00:00.000Z',
+          createdDate: '',
+          documentType: 'Article',
+          labsCollection: {
+            total: 3,
+          },
+          teamsCollection: {
+            items: [],
+          },
+        },
+      ];
+
+      graphqlResponse.teamsCollection!.items[0]!.linkedFrom!.researchOutputsCollection =
+        {
+          items: researchOutputs,
+        };
+      contentfulGraphqlClientMock.request.mockResolvedValueOnce(
+        graphqlResponse,
+      );
+
+      const result = await analyticsDataProvider.fetchTeamCollaboration({
+        take: 1,
+      });
+
+      expect(result).toEqual({
+        total: 1,
+        items: [
+          expect.objectContaining({
+            outputsCoProducedWithin: {
+              Article: 2,
+              Bioinformatics: 0,
+              Dataset: 0,
+              'Lab Resource': 0,
+              Protocol: 0,
+            },
+          }),
+        ],
+      });
+    });
+  });
+
+  describe('across teams', () => {});
+  test('Should return a count of 2 if client has two outputs of the same document type with multiple teams', async () => {
+    const graphqlResponse = getTeamCollaborationQuery();
+    const researchOutputs = [
```

Comment:
maybe you want to add an output which has only a single team to show it does not count?

Reply:
Added